### PR TITLE
fixed bridge networking

### PR DESCRIPTION
**Pull request by azeth**

### CODE REVIEW

File: dionysus.nix [hunk 1]
```diff
@@ -72,7 +77,8 @@
     ];
     hostId                           = "ce5e3a09";
     hostName                         = "dionysus";
-    interfaces.enp8s0.ipv4.addresses = [
+    interfaces.enp8s0.useDHCP = false;
```
Comment (jupblb) [inline dionysus.nix:80]: Breaks `dnsmasq`

**Review by jupblb — CHANGES_REQUESTED** [review]

**Review by jupblb — COMMENTED** [review]
This does not seem to be introducing any change. The network bridge serves exactly one physical device.

**Comment by jupblb** [comment]
Fixed in https://github.com/jupblb/nix-config/commit/545aeede03ac5ec042f08a8e67c4f6329b1d4894

**Comment by azeth** [comment]
Network bridge vmbr0 serves two devices: enp8s0 and the one created by libvirt when starting machine

File: dionysus.nix (hunk 1 above)
Comment (azeth) [reply dionysus.nix:80]: enp8s0 doesn't use DHCPCD (DHCP client daemon) and this line isn't actually used in previous config

**Pull request closed without merging**